pagopa/io-app
# feat: [IOPID-1504] Adopt the new DS to `IdpSuccessfulAuthentication`

## Short description
This PR addresses the redesign of the `IdpSuccessfulAuthentication` by adopting the new DS. 

> [!TIP]
> Review Privacy ✅
> [MP Event](https://pagopa.atlassian.net/browse/IOPID-1599) ✅
>
> [![Run e2e tests](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml/badge.svg?branch=IOPID-1504-ds-idp-successful-authentication)](https://github.com/pagopa/io-app/actions/workflows/test-e2e.yml)

<details><summary>🎦 Demo 🎦</summary>
<p>

| Android a11y | iOS a11y |
| - | - | 
| <video src="https://github.com/pagopa/io-app/assets/16268789/0f85984e-e337-4977-a041-2c153d2ef2b3" /> | <video src="https://github.com/pagopa/io-app/assets/16268789/81fe097e-c797-454d-932f-4e74593f2b44" /> |

</p>
</details> 

## List of Changes
- Adopt the new DS through the use of `OperationResultScreenContent`. This component lacks an automatic a11y focus, so it works well on iOS if the previous focus was approximately here or below the content title. In this case is just enough because we have preferred an **announcement** instead of setting the focus on the screen content title because the screen is show just a short period of time.


## How to test
- Test a login flow on both Android and iOS devices.
- Test a11y on both Android and iOS devices.
- Check if the [this MP event](https://pagopa.atlassian.net/browse/IOPID-1599) is correctly mapped.
